cypress/support/commands/common.js

```diff
@@ -608,3 +609,16 @@ Cypress.Commands.add('c_skipPasskeysV2', () => {
       }
     })
 })
+
+Cypress.Commands.add('c_fakeLinkPopUpCheck', () => {
+  cy.getCookie('website_status').then((cookie) => {
+    if (cookie && cookie.value === expectedCookieValue) {
```

Comment:
Can you try with this? Just for simplicity :)
```suggestion
    if (cookie?.value === expectedCookieValue) {
```